File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.cpp
class Solution {
public:
    vector<int> productExceptSelf(vector<int>& nums) {
        vector<int> new_nums(nums.size(), 1);
        for(int i = 0; i < nums.size(); i++) {
            for(int j = 0; j < nums.size(); j++) {
                if(i != j) {
                    new_nums[i] *= nums[j];
                }
            }
        }

        return new_nums;
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        seen = set(nums)
        count = 0

        for n in nums:
            if (n - 1) not in seen:
                length = 0
                while(n + length) in seen:
                    length += 1
                
                if length > count:
                    count = length
        

        return count


                
